src/components/CampaignPage.js

```diff
@@ -720,6 +735,16 @@ class CampaignPage extends Component {
                     modalButtonVariant: '#E63C36', waitToClose: false, tryAgainCC: true,
                     donationAmount: params.am
                 });
+            } else if(params.fp === 'pa'){
```

Comment:
here and other places, add space between `)` and `{`

Reply:
done in all files